client/app/components/Footer.jsx

```diff
@@ -1,27 +1,12 @@
 import React from 'react';
```

Comment:
Maybe delete the whole footer?

Reply:
Out of this scope :)

Reply:
It's just deleting it 🤷‍♂️

Between the help button and version in dropdown, there is very little use for the footer. We can do this in a follow up, if you prefer.

Reply:
No risk of course but I need to feel confident that it's justified by bringing it to the team.